app/AppKernel.php

```diff
@@ -41,4 +41,24 @@ public function registerContainerConfiguration(LoaderInterface $loader)
     {
         $loader->load(__DIR__.'/config/config_'.$this->getEnvironment().'.yml');
     }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @note Overridden path is to retain the pre-existing path structure and can be removed if moving to Symfony defaults
+     */
+    public function getCacheDir()
+    {
+        return $this->getProjectDir().'/app/cache/'.$this->environment;
```

Comment:
Just as the docblock notes, I added overrides for `getCacheDir()` and `getLogDir()` to keep those inside the `<project>/app` folder.  This can be dropped if you're good with moving these to the default `<project>/var` folder.